### DISCUSSION TITLE
Image dimensions without loading the image

### DISCUSSION THREAD

**Discussion by GulamY** [picoe/Eto#2030] [Q&A]
Hello!

Is there a way to read the dimensions of an image stored on disk without loading it? There are some platform specific mechanisms, I was hoping I could do it purely with Eto.

**Comment by cwensley** [comment]
Hey @GulamY,

No there aren't any specific API's for this, and it doesn't really make sense to add them for Eto specifically.  If you want to extract metadata from images without loading them there may be other ways to do that, such as with ImageSharp.
